demo/addons/godot-openxr/godot_openxr.gdnlib

```diff
@@ -9,10 +9,10 @@ reloadable=false
 
 Windows.64="res://addons/godot-openxr/bin/win64/libgodot_openxr.dll"
 X11.64="res://addons/godot-openxr/bin/linux/libgodot_openxr.so"
-Android.64="res://addons/godot-openxr/bin/android/libgodot_openxr.aar"
+Android.64="res://addons/godot-openxr/bin/android/libgodot_openxr.so"
 
```

Comment:
@m4gr3d Like we discussed on chat, I believe this should be .aar so the Godot export knows that this is the file that it needs to process, but during export we look into a way that the runtime knows the aar file extracts an so file and puts it in this place.

Reply:
This will require changes in Godot, so while we're still doing this the same way as with the quest plugin we'll leave this as `.so`

Reply:
After talking this through and further discussing the solution Fredia came up with, very happy with the end result.